README.md

```diff
@@ -1,318 +1,8 @@
 # Hazelcast Operator
```

Comment:
I remember we provided reference links to that README when working with Ted@IBM so those links can be broken now. I also remember I shared this this with some other people too.
https://github.com/hazelcast/hazelcast-operator#openshift-deployment-steps

Reply:
I see, hmm, then what do you suggest move it back to the main directory? And have a separate directory for Opensource?

In general, I think with the links provided they'll find their way.

Reply:
> In general, I think with the links provided they'll find their way. If we want to make the community use it, we need to make the opensource easy to install.

Ok. I am fine with that.